pages/agent/v3/windows.md.erb

```diff
@@ -13,16 +22,44 @@ The Buildkite Agent can be installed on both 64bit and 32bit editions of Windows
 SSH keys should be copied to (or generated into) in the user’s `.ssh` directory. For example, typing the following into Git Bash will generate a new private key which you can add to your source code host:
 
 ```bash
-$ mkdir -p ~/.ssh && cd ~/.ssh
 $ ssh-keygen -t rsa -b 4096 -C "build@myorg.com"
 ```
 
 See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details.
 
+## File Locations
+
+* Configuration: `C:\buildkite\buildkite-agent.cfg`
+* Hooks: `C:\buildkite\hooks`
+* Builds: `C:\buildkite\builds`
+* SSH keys: `%USERPROFILE%\.ssh`
+
+## Configuration
+
+The configuration file is located at `C:\buildkite\buildkite-agent.cfg`. See the [configuration documentation](/docs/agent/v3/configuration) for an explanation of each configuration setting.
+
+## Upgrading
+
+Simply rerun the install script.
```

Comment:
Can we remove the 'simply' here? It would be even more to the point as 'Rerun the install script' 😊

Reply:
Copy/pasted from our other docs, but agree this is a good change and will make it across the board.